pages/common/openttd.md

```diff
@@ -0,0 +1,28 @@
+# openttd
+
+> Open source clone of the Microprose game “Transport Tycoon Deluxe".
+> More information: <https://www.openttd.org>.
+
+- Load new/save game at start:
+
+`openttd -g {{path/to/file}}`
+
+- Start with a set resolution (eg. 1920x1200):
+
+`openttd -r {{resolution}}`
```

Comment:
```suggestion
- Start with the specified window dimensions:

`openttd -r {{1920x1200)}}`
```
Also, why not 1920x**1080**? Those are more common laptop dimensions, I think.

Reply:
Fair enough, I just pulled a resolution out thin air. Why window dimensions vs resolution though? :thinking:

Reply:
I'm not sure, looks like I made a mistake there.  Updated.